### ISSUE TITLE
API connection issue between Cloudstack 4.17.2 and whmcs

### ISSUE THREAD

**Issue by meisammaleki** [apache/cloudstack#7992]
we got this error on management log . list instances on cloudstack failed with connection times out and all action from whmcs server failed
its a major problem

##### ACTUAL RESULTS
2023-09-23 15:55:04,529 DEBUG [c.c.a.ApiServer] (qtp1750498848-424:ctx-277f0072 ctx-3fd47725) (logid:8d9ae664) CIDRs from which account 'Acct[b3eb3659-77d0-4559-9c79-d81b0ba5fe9e-WHMCS] -- Account {"id": 13, "name": "WHMCS"": "b3eb3659-77d0-4559-9c79-d81b0ba5fe9e"}' is allowed to perform API calls: 0.0.0.0/0,::/0
2023-09-23 15:55:04,549 DEBUG [c.c.a.d.ParamProcessWorker] (qtp1750498848-424:ctx-277f0072 ctx-3fd47725 ctx-17477c38) (logid:8d9ae664) Object entity uuid = 8d538294-a65e-47c0-a28b-cf1bb9a83dfd does not exist in the database
2023-09-23 15:55:04,549 INFO [c.c.a.ApiServer] (qtp1750498848-424:ctx-277f0072 ctx-3fd47725 ctx-17477c38) (logid:8d9ae664) Unable to execute API command queryasyncjobresult due to invalid value. Invalid parameter jobid val8294-a65e-47c0-a28b-cf1bb9a83dfd due to incorrect long value format, or entity does not exist or due to incorrect parameter annotation for the field in api cmd class.
2023-09-23 15:55:04,550 DEBUG [c.c.a.ApiServlet] (qtp1750498848-424:ctx-277f0072 ctx-3fd47725 ctx-17477c38) (logid:8d9ae664) ===END=== 80.75.0.10 -- GET jobid=8d538294-a65e-47c0-a28b-cf1bb9a83dfd&command=queryAsyncJobResuey=vNacmEAZP1kZluwMvwur2UtXWDDaTmj_qw6r7ZPYCzo85RQcsgIZkCvAnZAeCLN3hfJCnKcGutECQREJyu7pig&signatureversion=3&expires=2023-09-24T01%3A55%3A04%2B0330&response=json&signature=1%2BBFLe3g8TdsPCHeMz6EaH10%2BPU%3D
2023-09-23 15:55:04,664 DEBUG [c.c.a.ApiServlet] (qtp1750498848-424:ctx-e1c6c046) (logid:17e772e0) ===START=== 80.75.0.10 -- GET jobid=1d025991-279b-4e4b-8450-4e1273ba5a6e&command=queryAsyncJobResult&apiKey=vNacmEAZP1kZluUtXWDDaTmj_qw6r7ZPYCzo85RQcsgIZkCvAnZAeCLN3hfJCnKcGutECQREJyu7pig&signatureversion=3&expires=2023-09-24T01%3A55%3A04%2B0330&response=json&signature=gHEBa%2BoRB7WOcOTF6H%2BYUOuDXGo%3D
2023-09-23 15:55:04,679 DEBUG [c.c.a.ApiServer] (qtp1750498848-424:ctx-e1c6c046 ctx-86858329) (logid:17e772e0) CIDRs from which account 'Acct[b3eb3659-77d0-4559-9c79-d81b0ba5fe9e-WHMCS] -- Account {"id": 13, "name": "WHMCS"": "b3eb3659-77d0-4559-9c79-d81b0ba5fe9e"}' is allowed to perform API calls: 0.0.0.0/0,::/0
2023-09-23 15:55:04,699 DEBUG [c.c.a.d.ParamProcessWorker] (qtp1750498848-424:ctx-e1c6c046 ctx-86858329 ctx-eb2ee1b9) (logid:17e772e0) Object entity uuid = 1d025991-279b-4e4b-8450-4e1273ba5a6e does not exist in the database
2023-09-23 15:55:04,699 INFO [c.c.a.ApiServer] (qtp1750498848-424:ctx-e1c6c046 ctx-86858329 ctx-eb2ee1b9) (logid:17e772e0) Unable to execute API command queryasyncjobresult due to invalid value. Invalid parameter jobid val5991-279b-4e4b-8450-4e1273ba5a6e due to incorrect long value format, or entity does not exist or due to incorrect parameter annotation for the field in api cmd class.

**Comment by rajujith** [comment]
Hi @meisammaleki  do you see the issue when using cloudstack UI/ cloudmonkey?

**Comment by meisammaleki** [comment]
Yes web have issue on ACS UI . Instances section get time out and dosent show anything. piceture uploaded on this issue#7993

**Comment by rajujith** [comment]
@meisammaleki  can you share the steps to reproduce the issue?

**Comment by meisammaleki** [comment]
We did a version upgrade from 4.13 to 4.17 around 1 month ago. After the upgrade, ACS Works fine but the Compute/instances response was a little slow. but now it can't list instances at all. I think there is a problem with DB. I think the query stuck on VM stats table

**Comment by meisammaleki** [comment]
https://localhost/client/api/?command=listVirtualMachines&listall=true&dominid=&page=1&pageSize=2
list virtual machine API get 504 times out - The server did not respond

**Comment by harikrishna-patnala** [comment]
@meisammaleki if possible can you try restarting the management server. Log snippet does nt look like there is an issue with DB, but you can also try restarting the mysql service.

**Comment by JoaoJandre** [comment]
@DaanHoogland this seems like a environment issue. I'll be removing the milestone, if the issue is traced back to the source code, we can re-add the milestone.

**Comment by DaanHoogland** [comment]
@meisammaleki this issue went stale. Can you try the newest version and create a new issue if it still persists?

**Comment by meisammaleki** [comment]
@DaanHoogland I will try it in latest version
